### ISSUE TITLE
Nuxt-Quasar not working

### ISSUE THREAD

**Issue by R0N1n-dev** [Maiquu/nuxt-quasar#22]
### Checks

- [X] I have searched the existing issues for a similar bug and it does not exist!

### Environment

- Operating System: `Linux`
- Node Version:     `v16.17.0`
- Nuxt Version:     `3.1.2`
- Nitro Version:    `2.1.2`
- Package Manager:  `npm@8.15.0`
- Builder:          `vite`
- User Config:      `head`, `modules`, `components`
- Runtime Modules:  `nuxt-quasar-ui@1.3.0`
- Build Modules:    `-`

### Quasar version

2.11.9

### Browsers

Brave

### Description

On starting the dev server, following errors appear in :
1. The terminal:
----
ERROR  [unimport] failed to find "useHeadSafe" imported from "#imports"
----

2. The browser:

----
500
[vite-node] [plugin:nuxt:imports-transform] [VITE_ERROR] virtual:nuxt:./.nuxt/quasar-plugin.server.mjs

import { ref, computed, useHeadSafe } from "#imports"
import { defineNuxtPlugin } from "#app"
import Quasar from "quasar/src/vue-plugin.js"

import iconSet from "quasar/icon-set/material-icons"

export default defineNuxtPlugin((nuxt) => {
  const ssrContext = {
    req: nuxt.ssrContext.event.req,
    res: nuxt.ssrContext.event.res,
  }

  const bodyClasses = ref("")
  const htmlAttrs = ref("")

  const htmlAttrsRecord = computed(() => {
    return Object.fromEntries(
      htmlAttrs
        .split(" ")
        .map(attr => attr.split("="))
    )
  })

  useHeadSafe(computed(() => ({
    bodyAttrs: {
      class: bodyClasses.value
    }
  })))

  const NuxtPlugin = {
    install({ ssrContext }) {
      ssrContext._meta = new Proxy({}, {
        get(target, key) {
          if (key === "bodyClasses") {
            return bodyClasses.value
          } else if (key === "htmlAttrs") {
            return htmlAttrs.value
          } else {
            return target[key]
          }
        },
        set(target, key, value) {
          if (key === "bodyClasses") {
            bodyClasses.value = value
          } else if (key === "htmlAttrs") {
            htmlAttrs.value = value
          } else {
            target[key] = value
          }
          return true
        }
      })
    }
  }

  nuxt.vueApp.use(Quasar, {
    iconSet,
    plugins: {
      NuxtPlugin, 
    }
  }, ssrContext)
})

at virtual:nuxt:./.nuxt/quasar-plugin.server.mjs
----

### Reproduction steps

```bash
1. Install nuxt-quasar-ui in nuxt3 project according to instructions on module page
2. Run dev server
3. Open localhost:3000 in browser
```

**Comment by Maiquu** [comment]
Ah I was a bit eager to use `useHeadSafe`. If you upgrade nuxt to `^3.3.0`, it should be resolved. I'll probably make it fallback to `useHead` for earlier versions.

**Comment by Maiquu** [comment]
Fixed with #23

**Comment by R0N1n-dev** [comment]
Thank you.  Upgraded and it worked
